libkodimote/kodi.cpp

```diff
@@ -168,6 +168,8 @@ Kodi::Kodi(QObject *parent) :
     qmlRegisterType<KodiHost>(qmlUri, 1, 0, "KodiHost");
     qmlRegisterType<KodiDiscovery>(qmlUri, 1, 0, "KodiDiscovery");
 
+    qmlRegisterUncreatableType<Settings>("Kodi", 1, 0, "Settings", "Cannot create multiple settings. Use the \"settings\" context property intead");
```

Comment:
good catch. thanks. ok. I'll leave it be then in order not conflict with your branch.